Source/WebInspectorUI/UserInterface/Views/LogContentView.js

```diff
@@ -520,7 +520,7 @@ WI.LogContentView = class LogContentView extends WI.ContentView
 
     _previousMessageRepeatCountUpdated(event)
     {
-        if (!this._logViewController.updatePreviousMessageRepeatCount(event.data.count, event.data.timestamp))
+        if (!this._logViewController.updatePreviousMessageRepeatCount(event.data.target, event.data.count, event.data.timestamp))
```

Comment:
NIT: rather than repeating `event.data` i think we should extract it earlier
```js
let {target, count, timestamp} = event.data;
if (!this._logViewController.updatePreviousMessageRepeatCount(target, count, timestamp))
    return;
```